unique_by_attr.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python3
+"""
+    If we have some sequence of objects and want to remove items with the same attribute value
+    Python creates a dict, where keys are value if attribute (bar in our case), values are object of the sequence.
+    After that the dict is transformed back to list
+
+    Note: in result we save the last from repeating elements (item2 in our case)!
+"""
+
+
+class Foo():
+    pass
```

Comment:
I can, but the main idea of "bar" attribute is to show  which elements python will remove from secuence and why. So I suppose setting that attr for each item separately suits here pretty nice